NORTRIP_main_run.f90

```diff
@@ -25,16 +25,20 @@
 !
 !****************************************************************************
 
-    subroutine NORTRIP_main_run
-  
-    use NORTRIP_definitions
+subroutine NORTRIP_main_run
     
+    use NORTRIP_definitions
+    use NORTRIP_main_run_forecast
```

Comment:
This is for the case of "forecast analysis". Running in such mode, where forecast_hour is not 0, have not been well tested yet.